ui/src/app/utils/institutions.tsx

```diff
@@ -0,0 +1,21 @@
+import {AccountCreationOptions} from 'app/pages/login/account-creation/account-creation-options';
+import {InstitutionalRole, PublicInstitutionDetails} from 'generated/fetch';
+import {isBlank} from './index';
+
+export const getRoleOptions = (institutions: Array<PublicInstitutionDetails>, institutionShortName: string):
+    Array<{ label: string, value: InstitutionalRole }> => {
+  if (isBlank(institutionShortName)) {
+    return [];
+  }
+
+  const selectedOrgType = institutions.find(
```

Comment:
Do we have a standard around using native find (and other array functions), or utilize lodash find in these cases?
My personal preference is to keep things consistent with lodash, which also has other benefits like auto currying.

Also, a few nits - could this be changed to destructure out `organizationTypeEnum` (reassigning the name in the destructure). Could `shortName` be destructured out as well?